runtime/Cpp/runtime/src/tree/xpath/XPathLexer.cpp

```diff
@@ -61,7 +61,7 @@ void xpathLexerInitialize() {
       "STRING"
     }
   );
-  static const int32_t serializedATNSegment0[] = {
+  static const int32_t serializedATNSegment[] = {
```

Comment:
These could even be `static constexpr`. Probably sizeof() below will be smart enough to already know this at compile-time even with `const`, but with `constexpr` it doesn't have an excuse not to know.